src/resources/birthdayTimer.ts

```diff
@@ -0,0 +1,61 @@
+import { bdayDates, bdayTimes, bdayChannels } from '../slashcommands/Birthday';
+import cron from 'node-cron';
+import { Bot } from '../Bot';
+import { MessageEmbed, TextChannel, User } from 'discord.js';
+import Enmap from 'enmap';
+
+export let bdayCrons = new Enmap('bdayCrons');
+
+export async function birthdayTimer(guild: string, bot: Bot): Promise<void> {
+	let guildId = guild;
+	let infoString = bdayTimes.ensure(guild, '') as String;
+	console.log(infoString);
```

Comment:
We shouldn't be printing directly to console if we have access to bot.logger

Reply:
removed